### PR TITLE
Gateway coordinator 59

### PR DESCRIPTION
**Pull request by RafaelAPB**

### CODE REVIEW

File: packages/cactus-plugin-satp-hermes/README.md [hunk 1]
```diff
@@ -79,31 +85,31 @@ There are also defined the endpoints for the crash recovery procedure (there is
...
+Alice and Bob, in blockchains A and B, respectively, want to make a transfer of an asset from one to the other. Gateway A represents the gateway connected to Alice's blockchain. Gateway B represents the gateway connected to Bob's blockchain. Alice and Bob will run SATP, which will execute the transfer of the asset from blockchain A to blockchain B. The above endpoints will be called in sequence. Notice that the asset will first be locked on blockchain A and a proof is sent to the server-side. Afterward, the asset on the original blockchain is extinguished, followed by its regeneration on blockchain B.
 
 ## Running the tests
 
-[A test of the entire protocol with manual calls to the methods, i.e. without ledger connectors and Open API.](https://github.com/hyperledger/cactus/blob/2e94ef8d3b34449c7b4d48e37d81245851477a3e/packages/cactus-plugin-odap-hermes/src/test/typescript/integration/odap.test.ts)
```
Comment (AndreAugusto11) [inline packages/cactus-plugin-satp-hermes/README.md:86]: @RafaelAPB all of these should be odap-hermes. They point to a specific commit in which the package was not called this. Or we can rename them and point to a more recent

File: packages/cactus-plugin-satp-hermes/src/test/typescript/fabric-contracts/lock-asset/chaincode-typescript/package.json [hunk 2]
```diff
@@ -1,62 +0,0 @@
-{
```
Comment (AndreAugusto11) [inline packages/cactus-plugin-satp-hermes/src/test/typescript/fabric-contracts/lock-asset/chaincode-typescript/package.json:1]: @RafaelAPB was it on purpose? deleting the chaincode

**Review by petermetz — COMMENTED** [review]
@RafaelAPB @AndreAugusto11  I recommend keeping a single base branch if you can (I saw that the same branch name is also on Andre's fork when I was just there rebasing it onto Rafael's dev branch).

**Comment by AndreAugusto11** [comment]
> @RafaelAPB @AndreAugusto11 I recommend keeping a single base branch if you can (I saw that the same branch name is also on Andre's fork when I was just there rebasing it onto Rafael's dev branch).

Yes, my bad! I was doing some experiments and had to add some new commits to fix problems.
@RafaelAPB can you close this one pls, and we keep the other one? It has all of your changes and some additional fixes

**Comment by RafaelAPB** [comment]
@AndreAugusto11 closing this, thanks for all

**Pull request closed without merging**